gateway/1.4/modules/ROOT/pages/flex-gateway-k8-change-helm-settings.adoc

```diff
@@ -0,0 +1,231 @@
+
+. Change the `cpu` and `memory` values by using the `helm upgrade` command.
++
+.Syntax:
+----
+helm upgrade <release-name> <repository name>/<chart name> \
+-i --wait -n <namespace> --create-namespace \
+--set-file registration.content=registration.yaml \
+--set <key-to-update>=<new-value> 
+--set <next-key-to-update>=<new-value> 
+...
+----
++
+The `helm upgrade` command relies on the `registration.yaml` file but does not add your new settings to this file. 
++
+.Example:
+[source,helm]
+----
+helm upgrade ingress flex-gateway/flex-gateway \
+-i --wait -n gateway --create-namespace \
+--set-file registration.content=registration.yaml \
+--set resources.limits.cpu=1500m \
+--set resources.limits.memory=2048Mi \
+--set resources.requests.cpu=260m \
+--set resources.requests.memory=384Mi
+----
++
+When successful, the command returns output similar to this:
++
+----
+Release "ingress" has been upgraded. Happy Helming!
+NAME: ingress
+LAST DEPLOYED: Mon Mar 27 11:58:36 2023
+NAMESPACE: gateway
+STATUS: deployed
+REVISION: 3
+TEST SUITE: None
+----  
+
+. Using the new `REVISION` value, check the `resources` settings, for example:
++
+[source,helm]
+----
+helm get values --revision=3 ingress -n gateway
+----
++
+The output displays the new `resources` settings for the Pod:
++
+----
+USER-SUPPLIED VALUES:
+registration:
+  ...
+resources:
+  limits:
+    cpu: 1500m
+    memory: 2048Mi
+  requests:
+    cpu: 260m
+    memory: 384Mi
+----
++
+Note that the `resources` settings are not displayed through the `registration.yaml`.
+
+[[use-helm-f-option]]
+== Provide a File with Modified Settings for Your Helm Chart
+
+After following the steps in <<update-cpu-memory-example>>, use the `--f` option with `helm upgrade` to pass a file that contains new Helm chart settings for your Flex Gateway deployment. Alternatively, follow the procedure in <<use-helm-set-option>>.
+
+. From the command line, navigate to the directory that contains the YAML registration file for your gateway release, such as a `registration.yaml` for an `ingress` release.
++
+This file is generated when you register your gateway with Anypoint Platform in xref:flex-conn-reg-run.adoc[Connected Mode] or xref:flex-local-reg-run.adoc[Local Mode]. The file provides `registration.content` data for the `helm upgrade` step in this procedure. 
+
+. Generate a configuration file that contains your new settings.
++
+For example, use `echo` to create a configuration file named `my-config-file` that contains the new `cpu` and `memory` values:
++
+[source,echo]
+----
+echo '{resources.limits.cpu: 1500m, resources.limits.memory:2048Mi, resources.requests.cpu: 260m, resources.requests.memory: 384Mi}' \
+> my-config-file
+----
+
+. Pass the new values from your configuration file to your Helm chart, for example:
++
+.Syntax:
```

Comment:
I don't think the Syntax label is needed here - especially because there are no other labels used for step 3.